tests/api_tests/nodes/test_views.py

```diff
@@ -951,6 +952,92 @@ def test_node_children_list_does_not_include_unauthorized_projects(self):
+        # a little better
+        assert_equal(res.status_code, 403)
+
+        self.project.reload()
+        assert_equal(len(self.project.nodes), 0)
+
+    def test_creates_child_logged_in_owner(self):
+        res = self.app.post_json(self.url, self.child, auth=self.user.auth)
+        assert_equal(res.status_code, 201)
+        assert_equal(res.json['data']['title'], self.child['title'])
+        assert_equal(res.json['data']['description'], self.child['description'])
+        assert_equal(res.json['data']['category'], self.child['category'])
+
+        self.project.reload()
+        assert_equal(res.json['data']['id'], self.project.nodes[0]._id)
+
+    def test_creates_child_logged_in_write_contributor(self):
+        self.project.add_contributor(self.user_two, permissions=['read', 'write'], auth=Auth(self.user), save=True)
+        self.project.reload()
+
+        res = self.app.post_json(self.url, self.child, auth=self.user_two.auth)
+        assert_equal(res.status_code, 201)
+        assert_equal(res.json['data']['title'], self.child['title'])
+        assert_equal(res.json['data']['description'], self.child['description'])
+        assert_equal(res.json['data']['category'], self.child['category'])
+
+        self.project.reload()
+        assert_equal(res.json['data']['id'], self.project.nodes[0]._id)
+
+    def test_creates_child_logged_in_read_contributor(self):
+        self.project.add_contributor(self.user_two, permissions=['read'], auth=Auth(self.user), save=True)
+        self.project.reload()
+
+        res = self.app.post_json(self.url, self.child, auth=self.user_two.auth, expect_errors=True)
+        assert_equal(res.status_code, 403)
+
+        self.project.reload()
+        assert_equal(len(self.project.nodes), 0)
+
+    def test_creates_child_logged_in_non_contributor(self):
+        res = self.app.post_json(self.url, self.child, auth=self.user_two.auth, expect_errors=True)
+        assert_equal(res.status_code, 403)
+
+        self.project.reload()
+        assert_equal(len(self.project.nodes), 0)
+
+    def test_creates_child_creates_child_and_sanitizes_html_logged_in_owner(self):
+        title = '<em>Cool</em> <strong>Project</strong>'
+        description = 'An <script>alert("even cooler")</script> child'
+
+        res = self.app.post_json(self.url, {
+            'title': title,
+            'description': description,
+            'category': 'project',
+            'public': True,
+        }, auth=self.user.auth)
+        child_id = res.json['data']['id']
+        assert_equal(res.status_code, 201)
+        url = '/{}nodes/{}/'.format(API_BASE, child_id)
+
+        res = self.app.get(url, auth=self.user.auth)
+        assert_equal(res.json['data']['title'], strip_html(title))
+        assert_equal(res.json['data']['description'], strip_html(description))
+        assert_equal(res.json['data']['category'], 'project')
```

Comment:
Okay.  My reasoning is that we already tested for create child logged in owner so it wouldn't matter but I think either way it's not a big deal.  Will change for consistency sake I suppose